BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -178,6 +178,7 @@ - (instancetype)init
                                                                  APIBaseURL:_APIBaseURL
                                                              APIAuthBaseURL:_APIAuthBaseURL
                                                                queueManager:_queueManager];
+        _OAuth2Session.userAgentPrefix = self.userAgentPrefix;
```

Comment:
Do we expect to already have a user agent prefix at this point or do we expect this to get nil-ed?